app/components/webauthn_input_component.rb

```diff
@@ -36,4 +41,12 @@ def initial_hidden_tag_options
       { class: 'js' }
     end
   end
+
+  def show_desktop_ft_unlock_option?
+    if desktop_ft_unlock_option? && I18n.locale == :en
+      true
+    else
+      false
+    end
```

Comment:
Nit: Can simplify this a bit.

```suggestion
    desktop_ft_unlock_option? && I18n.locale == :en
```